spi/auth-spi/src/main/java/org/eclipse/edc/api/auth/spi/ParticipantPrincipal.java

```diff
@@ -0,0 +1,35 @@
+/*
+ *  Copyright (c) 2025 Metaform Systems, Inc.
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Metaform Systems, Inc. - initial API and implementation
+ *
+ */
+
+package org.eclipse.edc.api.auth.spi;
+
+import java.security.Principal;
+import java.util.Collection;
+import java.util.List;
+
+public record ParticipantPrincipal(String participantContextId, String role, String scope) implements Principal {
+
+    public static final String ROLE_ADMIN = "edcv-admin";
+    public static final String ROLE_PROVISIONER = "edcv-provisioner";
+    public static final String ROLE_PARTICIPANT = "edcv-participant";
```

Comment:
referring to a previous comment: we do have roles as constant, they could be used in controller class annotations

Reply:
done